src/utils/CheckYourAnswers/getCYARowForGroup.js

```diff
@@ -0,0 +1,51 @@
+import FormComponent from '../../components/FormComponent';
```

Comment:
Actually, this will need to import `React` because it's outputting jsx...
```suggestion
import React from 'react';
import FormComponent from '../../components/FormComponent';
```

Reply:
Added react import